spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -89,6 +89,8 @@ However, some Java types are not legal bean types :
 * A parameterized type that contains a wildcard type parameter is not a legal bean type.
 * An array type whose component type is not a legal bean type.
 
+If the set of bean types for a bean contains non-legal bean types, the container will ignore them during <<typesafe_resolution, typesafe resolution>>.
```

Comment:
I think that we should rather state that illegal bean types are not added to the set of bean types. Note that we may need to ignore these types not only during typesafe resolution, i.e. proxy class generation...

I would also add a short note to all relevant sections in chapter 3, e.g. "3.1.2. Bean types of a managed bean".